src/components/panel/panel.spec.js

```diff
@@ -91,6 +91,52 @@ describe('$mdPanel', function() {
     expect(PANEL_WRAPPER_CLASS).not.toHaveClass(HIDDEN_CLASS);
   });
 
+  it('should trap focus inside of the panel with trapFocus=true', function() {
```

Comment:
Can you rename this test? Something like "should create and clean up focus traps"; as the TODO mentions you can't actually test the focus behavior but the test description makes it sound as if you are.